src/main/scala/edu/ie3/simona/model/participant/evcs/EvcsModel.scala

```diff
@@ -514,13 +514,13 @@ final case class EvcsModel(
       modelState: EvcsState,
       data: EvcsRelevantData,
   ): ApparentPower =
-    throw new NotImplementedError("Use calculatePowerAndEvSoc() instead.")
+    throw new NotImplementedError("Use chargeEv() instead.")
```

Comment:
Not sure. Wouldn't we like to take the schedule that becomes active when charging the ev? Most certain it will be the same as of `calculateNewScheduling()` but couldn't there be some cases where this not happens?

Reply:
In the end, it doesn't matter that much because there's no direct equivalent to the methods above in EvcsModel.

Reply:
That's true, We could give both as possiblity to use instead?

Reply:
Yes why not